src/js/actions/authActions.js

```diff
@@ -0,0 +1,85 @@
+import fetch from 'fetch-everywhere';
```

Comment:
I'm not using CoffeeScript though? When you import a file from a module it doesn't need the extension. Unless you mean something else I'm not seeing?